Tests/Benchmarks/Benchmarks/ViewModel/MainPageViewModel.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Filters;
+using BenchmarkDotNet.Running;
+using Xamarin.Forms;
+
+namespace Benchmarks.ViewModel
+{
+    public class MainPageViewModel : BaseViewModel
+    {
+        string[] args;
+
+        private string benchmarkResults;
+
+        public string BenchmarkResults
+        {
+            get => benchmarkResults;
+            private set => SetProperty(ref benchmarkResults, value);
+        }
+
+        public ICommand RunBenchmarksCommand { get; private set; }
+
+        public MainPageViewModel(string[] args)
+        {
+            this.args = args;
+
+            InitCommands();
+        }
+
+        void InitCommands()
```

Comment:
This can be manually inlined in the ctor - not a ton of benefit from having a separate method for it + will allow us to remove the setter from `RunBenchmarksCommand`.